frontend/public/components/pdb/pdb.tsx

```diff
@@ -0,0 +1,262 @@
+import * as React from 'react';
+import { sortable } from '@patternfly/react-table';
+import * as classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import { PodDisruptionBudgetModel } from '../../models';
+import { useK8sWatchResource } from '../utils/k8s-watch-hook';
+import { useListPageFilter } from '../factory/ListPage/filter-hook';
+import ListPageFilter from '../factory/ListPage/ListPageFilter';
+import ListPageHeader from '../factory/ListPage/ListPageHeader';
+import ListPageBody from '../factory/ListPage/ListPageBody';
+import { ListPageCreateLink } from '../factory/ListPage/ListPageCreate';
+import { PodDisruptionBudgetKind, referenceForModel } from '../../module/k8s';
+import VirtualizedTable, { TableData } from '../factory/Table/VirtualizedTable';
+import { RowProps, TableColumn } from '@console/dynamic-plugin-sdk';
+import { useActiveColumns } from '../factory/Table/active-columns-hook';
+import {
+  Kebab,
+  ResourceLink,
+  Timestamp,
+  ResourceKebab,
+  ResourceSummary,
+  SectionHeading,
+  navFactory,
+  DetailsItem,
+  Selector,
+} from '../utils';
+import { DetailsPage } from '../factory';
+
+const tableColumnInfo = [
+  { className: '', id: 'name' },
+  { className: '', id: 'namespace' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-sm'), id: 'selector' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-md'), id: 'minAvailable' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-lg'), id: 'disruptionsAllowed' },
+  { className: classNames('pf-m-hidden', 'pf-m-visible-on-xl'), id: 'creationTimestamp' },
+  { className: Kebab.columnClass, id: '' },
+];
+
+const { common } = Kebab.factory;
+const menuActions = [...Kebab.getExtensionsActionsForKind(PodDisruptionBudgetModel), ...common];
```

Comment:
This seems to be a problem everywhere, but it looks like a race here. If the first page you visit is the PDB page, the plugins might not be loaded when this is evaluated.

We can ignore this in your PR, but do you mind opening a Bugzilla bug to look at this generally in console?